libraries/botbuilder-dialogs/src/choices/channel.ts

```diff
@@ -42,7 +42,7 @@ export function supportsCardActions(channelId: string, buttonCnt = 100): boolean
         case Channels.Facebook:
         case Channels.Skype:
         case Channels.Msteams:
-            return buttonCnt <= 3;
+            return buttonCnt <= 50;
```

Comment:
Here you are changing the count for the three channels (facebook, skype and msteams). We only have to change it for msteams.